### DISCUSSION TITLE
Drag & Drop : DragOver event not raised

### DISCUSSION THREAD

**Discussion by LaurentInSeattle** [AvaloniaUI/Avalonia#17238] [Q&A] [answered]
When dragging, the DragOver event is not raised on controls that have AllowDrop==true and no background.
After setting the background property to something almost transparent, like: 	Background="#01010101"  , it is back in business... 

Took me a long while to figure that out and I tend to think it is a bug...
 Or.. What is the rationale for such a behaviour ? 
 
 Thanks !

**Comment by stevemonaco** [comment, marked as answer]
By default, hit testing of (most?) controls fail when the `Background` is `null`. No hit test success = no pointer events. You can set to `Transparent` instead. Rationale should be WPF compatibility and performance.
